src/components/head-metadata/index.tsx

```diff
@@ -0,0 +1,47 @@
+import HashiHead from '@hashicorp/react-head'
+import { useCurrentProduct } from 'contexts'
+import { HeadMetadataProps } from './types'
+
+/**
+ * Builds up the the necessary meta tags for the site. Rendered in `_app`, where it receives `pageProps.metadata` as props
+ */
+export default function HeadMetadata(props: HeadMetadataProps) {
+  const { name: productName } = useCurrentProduct() ?? {}
```

Comment:
🙋🏻 Was there a page where `currentProduct` wasn't defined an issue came up here?

Reply:
The homepage is the main one I believe!